open-api.yaml

```diff
@@ -1652,6 +1691,15 @@ paths:
                       - price
                       - author
                       - title
+                    typoTolerance:
+                      enabled: true
+                      disableOnAttributes:
+                        - author
+                        - price
+                      disableOnWords:
+                        - Tolkien
```

Comment:
Oh! About `Tolkien` here or `disableOnWords` ?

Reply:
I bet the configuration field `disableOnWords` is. But my question is regarding the data inside of it 😬

Reply:
I think it cannot be case sensitive because internally the engine lowercase and deunicode all strings in documents. @ManyTheFish Correct me if I'm wrong 🤓

I will add it!